ee/clickhouse/queries/experiments/trend_experiment_result.py

```diff
@@ -469,6 +476,38 @@ def calculate_p_value(control_variant: Variant, test_variants: list[Variant]) ->
     )
 
 
+def calculate_credible_intervals(variants, lower_bound=0.025, upper_bound=0.975):
```

Comment:
[nit] should these defaults be global constants at the top of the file rather than defined in the function?  I could go either way on this, just figured I'd make a note.

Reply:
we need to make the confidence interval configurable in the future, so figured I would pass these as arguments, but good point 👍